app/src/InnovationWeek.php

```diff
@@ -1,26 +1,41 @@
 <?php
 
 use MaximeRainville\SilverstripeReact\ReactAdmin;
-use PharIo\Manifest\Requirement;
+use SilverStripe\Control\Controller;
 use SilverStripe\Core\Manifest\ModuleResourceLoader;
-use SilverStripe\View\Requirements;
 
 class InnovationWeek extends ReactAdmin
 {
     private static $url_segment = 'innovation';
 
     private static $menu_title = 'Innovation Week';
 
-    public function getProps(): array
```

Comment:
The Router bootstrapping doesn't allow use to pass props or a specific component name to the front end since there's potential many components needed to render your application based on the route.